src/pages/audio/_id.vue

```diff
@@ -1,9 +1,17 @@
 <template>
-  <main class="relative">
-    <div v-if="backToSearchPath" class="w-full p-2">
-      <VBackToSearchResultsLink :path="backToSearchPath" />
+  <main>
+    <div
+      v-if="backToSearchPath"
+      class="w-full py-2"
+      :class="{ 'px-2 md:px-6': !isNewHeaderEnabled }"
+    >
+      <VBackToSearchResultsLink :href="backToSearchPath" />
+    </div>
+
+    <!-- Negative margin compensates for the `!px-6` padding on <Nuxt/> component. -->
```

Comment:
I'm not sure why we have `!px-6` on the `<Nuxt>` component. Can we remove it instead of adding extra `-mx-6` here. This will also fix the current excessive padding on the mobile: It should have 24px horizontal padding, and now has twice that.

Reply:
@obulat I was hoping to ask you about it 😄 because you added it in a previous commit. We could remove that and let the pages handle any horizontal margin themselves.

Reply:
I've added it here https://github.com/WordPress/openverse-frontend/pull/1875, but I really don't understand why 😕 Please go ahead and remove it :)